src/Migrations/M034MigrateSplit2ColumnBlock.php

```diff
@@ -0,0 +1,203 @@
+ */
+class M034MigrateSplit2ColumnBlock extends MigrationScript
+{
+    /**
+     * Extract split-two-columns block from page/posts and transform it into columns block.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    public static function execute(MigrationRecord $record): void
+    {
+        // Get the list of posts using split-two-columns block.
+        $posts = Utils\Functions::get_posts_using_specific_block(
+            Utils\Constants::BLOCK_SPLIT_TWO_COLUMNS,
+            Utils\Constants::ALL_POST_TYPES
+        );
+
+        // If there are no posts, abort.
+        if (!$posts) {
+            return;
+        }
+
+        $parser = new WP_Block_Parser();
+
+        echo "Split 2 Columns block migration in progress...\n"; // phpcs:ignore
+
+        foreach ($posts as $post) {
+            if (empty($post->post_content)) {
+                continue;
+            }
+
+            echo 'Parsing post ', $post->ID, "\n"; // phpcs:ignore
+            $result = false;
+
+            // Go through the post blocks to find the planet4-blocks/split-two-columns one.
+            $blocks = $parser->parse($post->post_content);
+
+            foreach ($blocks as &$block) {
+                // Skip other blocks.
+                if (! isset($block['blockName']) || $block['blockName'] !== Utils\Constants::BLOCK_SPLIT_TWO_COLUMNS) {
+                    continue;
+                }
+
+                $block = self::get_transform_block($block);
+            }
+
+            // Unset the reference to prevent potential issues.
+            unset($block);
+
+            // Serialize the blocks content.
+            $new_content = serialize_blocks($blocks);
+
+            if ($post->post_content !== $new_content) {
+                $post_update = array(
+                    'ID' => $post->ID,
+                    'post_content' => $new_content,
+                );
+
+                try {
+                    // Update the post with the replaced blocks.
+                    wp_update_post($post_update);
+                    $result = true;
+                } catch (\Throwable $e) {
+                    echo 'Error on post ', $post->ID, "\n";
+                    echo $e->getMessage(), "\n";
+                }
+            }
+
+            echo $result
+                ? "Migration successful\n"
+                : "Migration wasn't executed\n"; // phpcs:ignore
+        }
+    }
+    // phpcs:enable SlevomatCodingStandard.Functions.UnusedParameter
+
+    /**
+     * Transform a block attrs into columns block.
+     *
+     * @param array $block - A block data array.
+     * @return array - The transformed block.
+     * phpcs:disable Generic.Files.LineLength.MaxExceeded
+     */
+    private static function get_transform_block(array $block): array
```

Comment:
@sagarsdeshmukh 
This is just a personal preference, so please ignore this message if you disagree.
IMO, using WordPress Block Code makes the code harder to read, debug, and maintain.
The same result can be achieved with PHP arrays (check it below).
I know, the extension of the code increases, but by using arrays we can take advantage of reusable functions or values pulled from the constants file:

<details>
  <summary>Click to expand</summary>

```php
$block = array(
                    'blockName' => 'core/columns',
                    'attrs' => array(),
                    'innerBlocks' => array(
                        array(
                            'blockName' => 'core/column',
                            'attrs' => array(
                                'verticalAlignment' => 'center',
                            ),
                            'innerBlocks' => array(
                                array(
                                    'blockName' => 'core/media-text',
                                    'attrs' => array(
                                        'mediaId' => 350,
                                        'mediaLink' => 'http://www.planet4.test/get-informed/people/tavolo-community-school-in-papua-new-guinea/',
                                        'mediaType' => 'image',
                                    ),
                                    'innerBlocks' => array(
                                        array(
                                            'blockName' => 'core/heading',
                                            'attrs' => array(
                                                'level' => 3,
                                            ),
                                            'innerBlocks' => array(),
                                            'innerHTML' => '<h3 class="wp-block-heading">People</h3>',
                                            'innerContent' => array(
                                                '<h3 class="wp-block-heading">People</h3>',
                                            ),
                                        ),
                                        array(
                                            'blockName' => 'core/paragraph',
                                            'attrs' => array(),
                                            'innerBlocks' => array(),
                                            'innerHTML' => '<p></p>',
                                            'innerContent' => array('<p></p>'),
                                        ),
                                        array(
                                            'blockName' => 'core/paragraph',
                                            'attrs' => array(),
                                            'innerBlocks' => array(),
                                            'innerHTML' => '<p><a href="http://www.planet4.test/get-informed/people/">Learn more about this issue</a></p>',
                                            'innerContent' => array('<p><a href="http://www.planet4.test/get-informed/people/">Learn more about this issue</a></p>'),
                                        ),
                                    ),
                                    'innerHTML' => '<div class="wp-block-media-text is-stacked-on-mobile"><figure class="wp-block-media-text__media"><img src="http://www.planet4.test/wp-content/uploads/2018/10/46697a18-people-6aa59e55-gp04o3g_pressmedia.jpg" alt="" class="wp-image-350 size-full"/></figure><div class="wp-block-media-text__content"></div></div>',
                                    'innerContent' => array(
                                        '<div class="wp-block-media-text is-stacked-on-mobile"><figure class="wp-block-media-text__media"><img src="http://www.planet4.test/wp-content/uploads/2018/10/46697a18-people-6aa59e55-gp04o3g_pressmedia.jpg" alt="" class="wp-image-350 size-full"/></figure><div class="wp-block-media-text__content">',
                                        '',
                                        '',
                                        '',
                                        '',
                                        '</div></div>',
                                    ),
                                ),
                            ),
                            'innerHTML' => '<div class="wp-block-column is-vertically-aligned-center"></div>',
                            'innerContent' => array(
                                '<div class="wp-block-column is-vertically-aligned-center">',
                                '',
                                '</div>',
                            ),
                        ),
                        array(
                            'blockName' => 'core/column',
                            'attrs' => array(),
                            'innerBlocks' => array(
                                array(
                                    'blockName' => 'core/media-text',
                                    'attrs' => array(
                                        'mediaId' => 347,
                                        'mediaLink' => 'http://www.planet4.test/consumption-f7fb64f4-gp0str6il_pressmedia/',
                                        'mediaType' => 'image',
                                    ),
                                    'innerBlocks' => array(
                                        array(
                                            'blockName' => 'core/heading',
                                            'attrs' => array(
                                                'level' => 3,
                                            ),
                                            'innerBlocks' => array(),
                                            'innerHTML' => '<h3 class="wp-block-heading">#Consumption</h3>',
                                            'innerContent' => array('<h3 class="wp-block-heading">#Consumption</h3>'),
                                        ),
                                        array(
                                            'blockName' => 'core/paragraph',
                                            'attrs' => array(),
                                            'innerBlocks' => array(),
                                            'innerHTML' => '<p>Buy. Use. Throw away. Buy more. Fast food, fast fashion, single use&hellip;</p>',
                                            'innerContent' => array('<p>Buy. Use. Throw away. Buy more. Fast food, fast fashion, single use&hellip;</p>'),
                                        ),
                                        array(
                                            'blockName' => 'core/buttons',
                                            'attrs' => array(),
                                            'innerBlocks' => array(
                                                array(
                                                    'blockName' => 'core/button',
                                                    'attrs' => array(
                                                        'className' => 'is-style-cta',
                                                    ),
                                                    'innerBlocks' => array(),
                                                    'innerHTML' => '<div class="wp-block-button is-style-cta"><a class="wp-block-button__link wp-element-button" href="http://www.planet4.test/tag/consumption/">Get involved</a></div>',
                                                    'innerContent' => array('<div class="wp-block-button is-style-cta"><a class="wp-block-button__link wp-element-button" href="http://www.planet4.test/tag/consumption/">Get involved</a></div>'),
                                                ),
                                            ),
                                            'innerHTML' => '<div class="wp-block-buttons"></div>',
                                            'innerContent' => array(
                                                '<div class="wp-block-buttons">',
                                                '',
                                                '</div>',
                                            ),
                                        ),
                                    ),
                                    'innerHTML' => '<div class="wp-block-media-text is-stacked-on-mobile"><figure class="wp-block-media-text__media"><img src="http://www.planet4.test/wp-content/uploads/2018/10/d9f1b57c-consumption-f7fb64f4-gp0str6il_pressmedia.jpg" alt="" class="wp-image-347 size-full"/></figure><div class="wp-block-media-text__content"></div></div>',
                                    'innerContent' => array(
                                        '<div class="wp-block-media-text is-stacked-on-mobile"><figure class="wp-block-media-text__media"><img src="http://www.planet4.test/wp-content/uploads/2018/10/d9f1b57c-consumption-f7fb64f4-gp0str6il_pressmedia.jpg" alt="" class="wp-image-347 size-full"/></figure><div class="wp-block-media-text__content">',
                                        '',
                                        '',
                                        '',
                                        '',
                                        '</div></div>',
                                    ),
                                ),
                            ),
                            'innerHTML' => '<div class="wp-block-column"></div>',
                            'innerContent' => array(
                                '<div class="wp-block-column">',
                                '',
                                '</div>',
                            ),
                        ),
                    ),
                    'innerHTML' => '<div class="wp-block-columns"></div>',
                    'innerContent' => array(
                        '<div class="wp-block-columns">',
                        '',
                        '',
                        '',
                        '',
                        '</div>',
                    ),
                );
                
            }
```

</details>